room/runtime/src/main/java/androidx/room/RoomDatabase.java

```diff
@@ -1012,6 +1058,21 @@ private static boolean isLowRamDevice(@NonNull ActivityManager activityManager)
             return this;
         }
 
+        /**
+         * Adds a {@link TypeConverterFactory} to this database.
+         *
+         * @param factory The factory.
+         * @return This {@link Builder} instance.
+         */
+        @NonNull
+        public Builder<T> addTypeConverterFactory(@NonNull TypeConverterFactory factory) {
+            if(mTypeConverterFactories == null) {
```

Comment:
nit: missing space between `if` and `(`

Reply:
Weird, I ran `./gradlew buildOnServer ktlint` before pushing. It doesn't detect this.

Reply:
@dlam, does `./gradlew ktlint` perform formatting or is it just checking? I'm also not sure how strict is it with unconventional spaces...

Reply:
I mean I ran `./gradlew buildOnServer ktlint` as suggested here: https://github.com/androidx/androidx/pull/73
and expected it to fail if there is something wrong. I fixed a lot of formatting issues using this approach before and I believe it detected some missing spaces too. Not sure why it doesn't work in this case. I've just ran it again and got no errors.

Reply:
ktlint just checks, ktformat applies the fix

Reply:
we also normally run like three different kinds of lint as part of bOS, but I feel ktlint shouldve caught this.